File: calculator.py
#!/usr/bin/env python3
"""Add two numbers passed as command-line arguments."""

import sys


def main() -> int:
    if len(sys.argv) != 3:
        print("Usage: python3 calculator.py <A> <B>")
        return 1

    a = int(sys.argv[1])
    b = int(sys.argv[2])
    print(a + b)
    return 0


if __name__ == "__main__":
    raise SystemExit(main())

File: Calculator.java
public class Calculator {
    public static void main(String[] args) {
        if (args.length != 2) {
            System.out.println("Usage: java Calculator <A> <B>");
            return;
        }

        int a = Integer.parseInt(args[0]);
        int b = Integer.parseInt(args[1]);
        System.out.println(a + b);
    }
}
